### ISSUE TITLE
core-scroll: hasAttribute exception

### ISSUE THREAD

**Issue by laat** [nrkno/core-components#104]
I am getting this error in production:
```
TypeError: Object doesn't support property or method 'hasAttribute'
   at Anonymous function (https://tv.nrk.no/_static/serie/main.74fbcc73a21039e80a6d.js:1:421338)
```

Which is this line:
https://github.com/nrkno/core-components/blob/7377ae4add46ac33355b1021c120c86d8615ca3a/packages/core-scroll/core-scroll.js#L53

Spec:
* os: Windows 10
* Browser: Edge 17.17134

I am unable to reproduce it locally. Any ideas of how we can end up with an error like this?

**Comment by laat** [comment]
I'm also getting errors on the exact same line in Windows 10, Chrome 60.0

```
TypeError: t.hasAttribute is not a function
    at HTMLDocument.<anonymous> (https://tv.nrk.no/_static/serie/main.74fbcc73a21039e80a6d.js:1:421361)
```

**Comment by eirikbacker** [comment]
I see the problem - interesting that event.target can be document.. Where is the cursor while dragging? :)

**Comment by laat** [comment]
It is from the production log, so I do not know =/

**Comment by eirikbacker** [comment]
Closed by #110
